src/django/api/models/state.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.contrib.gis.db import models as gis_models
+
+__all__ = ["State"]
+
+
+class State(models.Model):
+    id = models.CharField(max_length=2, primary_key=True, unique=True)
```

Comment:
Prefer using `django.contrib.postgres.fields.CICharField` for a case-insensitive field that will match "NC", "nc", and any other combination.

Reply:
Nice, will implement! Will also get rid of the `clean()` method to uppercase that field since that will no longer be needed